src/BlazorDB/wwwroot/blazorDB.js

```diff
@@ -205,5 +285,136 @@ window.blazorDB = {
                 reject(e);
             });
         });
+    },
+    // Assumption. Only one schema update per version update
+    // Will allow multiple updates after proof of concept
+    // This function purposefully DOES NOT return a promise. This is because .upgrade() DOES NOT return a promise.
```

Comment:
Just noting this assumption.

I'm okay for a round 1 draft too, it's just one update per version and we just error if `storeSchemeUpgrades.length > 1` to let the user know.